resurfemg/postprocessing/features.py

```diff
@@ -0,0 +1,623 @@
+import collections
+import math
+import warnings
+import scipy
+from scipy.fft import fft, fftfreq
+from scipy.signal import savgol_filter
+from scipy.stats import entropy
+import matplotlib.pyplot as plt
+import numpy as np
+import logging
+from ..preprocessing.envelope import running_smoother
+from ..helper_functions.helper_functions import delay_embedding
```

Comment:
..helper_functions.helper_functions does not yet exist in this branch. Either merge the dev-helper_functions or dev branch into this one, or leave this line to the person making /helper_functions/helper_functions